app/views/questions/_answer.html.erb

```diff
@@ -32,7 +32,7 @@
     <hr />
     <div style="padding: 0px 20px 10px 20px;">
     <% if current_user && current_user.uid == answer.node.uid || answer.accepted %>
-      <a <% if current_user && current_user.uid == answer.node.uid %> href="/answers/accept/<%= answer.id %>" <% end %> data-remote="true" id="answer-<%= answer.id %>-accept" class="answer-accept btn btn-sm <% if answer.accepted %>btn-success<% else %>btn-default<% end%>"><% if answer.accepted %>Accepted<% else %>Accept this answer<% end %></a>
+      <a <% if current_user && current_user.uid == answer.node.uid %> href="/answers/accept/<%= answer.id %>" <% end %> data-remote="true" id="answer-<%= answer.id %>-accept" class="answer-accept btn btn-sm <% if answer.accepted %>btn-success<% else %>btn-default<% end%>"><% if answer.accepted %><p class="alert alert-success">Your answer was accepted! If it would be helpful, consider expanding on this answer by <a href="/post?template=activity,tags=response:<question-nid>">posting a step-by-step activity</a></p><% else %>Accept this answer<% end %></a>
```

Comment:
Hi, thanks for the PR!

Here, I think it'll be easier if you break it up into multiple lines; what's happening is we don't really want to delete the existing text, but to add the `<div class="alert...` onto its own line. 

And I think something else is going on too -- this conditional checks if the current user is the same as the question asker -- `answer.node.uid` -- but we want to know if it's the same as the **answerer** -- that'd be, I think, `answer.author.uid` according to this file: https://github.com/publiclab/plots2/blob/master/app/models/answer.rb

And as to where, I think we could display the alert right above the horizontal rule (`<hr />`) on this line: https://github.com/500swapnil/plots2/blob/21b7823534b62e9564839f15c9e2be6356efeab3/app/views/questions/_answer.html.erb#L32

Take a look at a page with answers, does that placement make sense to you? 

https://publiclab.org/questions/dhale2/12-01-2016/spectral-workbench-recording-time

Screenshot:

![screenshot 2016-12-03 at 5 50 09 pm](https://cloud.githubusercontent.com/assets/24359/20862743/559d2cc2-b981-11e6-969a-bd53a25ae082.png)